### ISSUE TITLE
Beet import failing - OSError: [WinError 17]

### ISSUE THREAD

**Issue by elit3ge** [beetbox/beets#4252]
As per the title, import a bunch of albums and it keeps on failing on this, run the script again and it works.

```
Traceback (most recent call last):
  File "C:\Python3\lib\site-packages\beets\util\__init__.py", line 496, in move
    os.replace(path, dest)
OSError: [WinError 17] The system cannot move the file to a different disk drive: '\\\\?\\C:\\Users\\Admin\\AppData\\Local\\Temp\\tmpn96odgj6.jpg' -> '\\\\?\\G:\\Music\\Pink Floyd\\The Screaming Abdabs_ Quebec City, 10 Nov 1971 [2021]\\cover.jpg'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Python3\Scripts\beet-script.py", line 33, in <module>
    sys.exit(load_entry_point('beets==1.6.0', 'console_scripts', 'beet')())
  File "C:\Python3\lib\site-packages\beets\ui\__init__.py", line 1285, in main
    _raw_main(args)
  File "C:\Python3\lib\site-packages\beets\ui\__init__.py", line 1272, in _raw_main
    subcommand.func(lib, suboptions, subargs)
  File "C:\Python3\lib\site-packages\beets\ui\commands.py", line 973, in import_func
    import_files(lib, paths, query)
  File "C:\Python3\lib\site-packages\beets\ui\commands.py", line 943, in import_files
    session.run()
  File "C:\Python3\lib\site-packages\beets\importer.py", line 340, in run
    pl.run_parallel(QUEUE_SIZE)
  File "C:\Python3\lib\site-packages\beets\util\pipeline.py", line 446, in run_parallel
    raise exc_info[1].with_traceback(exc_info[2])
  File "C:\Python3\lib\site-packages\beets\util\pipeline.py", line 358, in run
    self.coro.send(msg)
  File "C:\Python3\lib\site-packages\beets\util\pipeline.py", line 170, in coro
    task = func(*(args + (task,)))
  File "C:\Python3\lib\site-packages\beets\importer.py", line 1566, in manipulate_files
    task.manipulate_files(
  File "C:\Python3\lib\site-packages\beets\importer.py", line 766, in manipulate_files
    plugins.send('import_task_files', session=session, task=self)
  File "C:\Python3\lib\site-packages\beets\plugins.py", line 488, in send
    result = handler(**arguments)
  File "C:\Python3\lib\site-packages\beets\plugins.py", line 145, in wrapper
    return func(*args, **kwargs)
  File "C:\Python3\lib\site-packages\beetsplug\fetchart.py", line 1063, in assign_art
    self._set_art(task.album, candidate, not self.src_removed)
  File "C:\Python3\lib\site-packages\beetsplug\fetchart.py", line 1048, in _set_art
    album.set_art(candidate.path, delete)
  File "C:\Python3\lib\site-packages\beets\library.py", line 1317, in set_art
    util.move(path, artdest)
  File "C:\Python3\lib\site-packages\beets\util\__init__.py", line 499, in move
    prefix=py3_path(b'.' + os.path.basename(dest)),
TypeError: can't concat str to bytes
```

**Comment by jackwilsdon** [comment]
This should be fixed by #4209. To try it out you can either wait for the next release or [run the latest source version of beets](https://beets.readthedocs.io/en/latest/faq.html#run-the-latest-source-version-of-beets).

**Comment by MinchinWeb** [comment]
Ran into this issue. Running the source version seems to fix this. Looking forward to the 1.7 release :)
